### PR TITLE
Adjustments in ursacli

### PR DESCRIPTION
**Pull request by icedevml**
* Implied non-interactive (quiet) mode when input is not a TTY (e.g. `echo 'status' | ursacli`)
* Additional `-j` command line switch to turn off output postprocessing in the client (and always get the raw JSON response from server)
* By default, `select` queries will return a simple list of files (newline separated text)

### CODE REVIEW

File: src/Client.cpp [hunk 1]
```diff
@@ -109,12 +109,17 @@ void UrsaClient::recv_res(zmq::socket_t &socket) {
...
-                spdlog::error(res["error"]["message"].get<std::string>());
-            } else {
-                std::cout << res.dump(4) << std::endl;
+        if (this->force_json) {
+            std::cout << res.dump(4) << std::endl;
+            return;
+        }
+
+        if (res["type"] == "select") {
+            for (auto &res : res["result"]["files"]) {
```
Comment (msm-code) [inline src/Client.cpp:118]: it's correct, but maybe change the variable name to avoid shadowing :thinking: (for example to `file`)

Comment (icedevml) [reply src/Client.cpp]: :thinking: didn't even realized previously, fixed

File: src/Client.h [hunk 2]
```diff
@@ -13,6 +13,7 @@ class UrsaClient {
     std::string server_addr;
     std::string db_command;
     bool quiet_mode;
+    bool force_json;
```
Comment (chivay) [inline src/Client.h:16]: maybe `raw_json` instead of `force_json` ?

File: src/Client.cpp [hunk 3]
```diff
@@ -194,23 +200,28 @@ static void print_usage(const char *arg0) {
...
     spdlog::info(
         "    [-q]               silent mode, dump only command output");
+    spdlog::info("    [-j]               force JSON output everywhere");
 }
 
 int main(int argc, char *argv[]) {
     std::string server_addr = "tcp://localhost:9281";
     std::string db_command = "";
-    bool quiet_mode = false;
+    bool quiet_mode = !isatty(0);
```
Comment (chivay) [inline src/Client.cpp:209]: I would rephrase that into `is_interactive` or something like this

File: src/Client.cpp [hunk 4]
```diff
@@ -109,12 +109,17 @@ void UrsaClient::recv_res(zmq::socket_t &socket) {
...
-                spdlog::error(res["error"]["message"].get<std::string>());
-            } else {
-                std::cout << res.dump(4) << std::endl;
+        if (this->force_json) {
+            std::cout << res.dump(4) << std::endl;
+            return;
+        }
+
+        if (res["type"] == "select") {
+            for (auto &file : res["result"]["files"]) {
```
Comment (msm-code) [inline src/Client.cpp:118]: `const auto`?
